inst/examples/dashboard/about.R

```diff
@@ -0,0 +1,179 @@
+          professional look - rich set of components easily usable in Shiny -
+          fast speed of development that Shiny is famous for. As Fluent UI is
+          built in React, shiny.fluent is based on another package called
+          shiny.react, which allows for using React libraries in Shiny."
+        )
+      )
+    )
+  )
+)
+
+shiny_react_card <- div(
+  class = "card ms-depth-8",
+  Stack(
+    tokens = list(childrenGap = 10),
+    a(
+      href = "https://appsilon.github.io/shiny.react/",
+      span(class = "ms-font-su ms-fontColor-themePrimary", "shiny.react")
+    ),
+    Stack(
+      wrap = FALSE,
+      horizontal = TRUE,
+      horizontalAlign = TRUE,
+      verticalFill = TRUE,
+      tokens = list(childrenGap = 12),
+      a(
+        href = "https://appsilon.github.io/shiny.react/",
+        Image(
+          src = "img/shiny-react.png",
+          width = "120px",
+          height = "120px"
+        )
+      ),
+      Stack(
+        tokens = list(childrenGap = 5),
+        Text(
+          variant = "large", "handshake between React and Shiny",
+          block = TRUE
+        ),
+        Text(
+          "Most of the shiny apps are build directly in Shiny
+          without using any JS library. React being the Most popular one
+          becomes one of such candidate.
+          This R package enables using React in Shiny apps and is
+          used e.g. by the shiny.fluent package. It contains R and JS
+          code which is independent from the React library
+          (e.g. Fluent UI) that is being wrapped."
+        )
+      )
+    )
+  )
+)
+
+shiny_router_card <- div(
+  class = "card ms-depth-8",
+  Stack(
+    tokens = list(childrenGap = 10),
+    a(
+      href = "https://appsilon.github.io/shiny.router/",
+      span(class = "ms-font-su ms-fontColor-themePrimary", "shiny.router")
+    ),
+    Stack(
+      wrap = FALSE,
+      horizontal = TRUE,
+      horizontalAlign = TRUE,
+      verticalFill = TRUE,
+      tokens = list(childrenGap = 12),
+      a(
+        href = "https://appsilon.github.io/shiny.router/",
+        Image(
+          src = "img/shiny-router.png",
+          width = "120px",
+          height = "120px"
+        )
+      ),
+      Stack(
+        tokens = list(childrenGap = 5),
+        Text(
+          variant = "large", "Create Shiny like a MultiPage App",
+          block = TRUE
+        ),
+        Text(
+          " With Shiny Router we can actually create a shiny App That
+          behaves like a multipage App. Where you can direct user from
+          one point to another through using hrefs and hyperlinks.
+          This is something we never had in shiny and you might have
+          found yourself stuck in a project where people want to share
+          URL of only one perticular Page. With Shiny Router we try to
+          overcome these challenges in a shiny App. Try the package
+          and Give us your feedback So that we can grow together..."
```

Comment:
The "single-page app" and "multi-page app" are technical terms with specific meaning. In fact all Shiny apps are single-page apps (only one request is made at the beginning to load the page, which is then dynamically modified) and shiny.router doesn't change that.
```suggestion
          "With shiny.router you can introduce routing to your Shiny application:
          use the URL to direct user from one view to to another.
          This not only simplifies the implementation
          but also allows the users to bookmark or share the URL of a specific view."
```

Reply:
THanks for the explanation added the text as it is.